scripts/upgrade

```diff
@@ -173,7 +195,7 @@ pushd $final_path/$app/js
     ynh_script_progression --message="Building NodeJS..."
     ynh_use_nodejs    
     sudo -u $app env PATH=$PATH yarn install
-    sudo -u $app env PATH=$PATH NODE_BUILD_MEMORY=1024 yarn run build
+    sudo -u $app env PATH=$PATH yarn run build
```

Comment:
Eeeeh, why remove this option ? It sounded like a nice useful tweak ?

Reply:
I tryed that parameter to solve package_check upgrade test but later, when adding progression fo Nodejs and elixir build, I've seen that upgrade check failed cause of Elixir...